### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by gavlig** [gavlig/kodiki#3] [Announcements]
Github discussions are enabled in experimental mode to see if it makes sense

**Comment by ginxxxxx** [comment]
I really love your project.
I feel that the bevy engine's graphics can help visualize the "debugger" features of an editor/IDE

**Reply by gavlig** [reply]
Thank you! Yep, the intent is to visualize everything as much as possible utilizing 3d, shaders and all that. There is still a lot of work left to achieve the MVP level of functionality, but we'll get there eventually :) Cheers!
